content/blog/2019-year-at-a-glance/index.md

```diff
@@ -43,7 +43,7 @@ It's impossible to highlight everything, but here are some of our favorite miles
 
 **Coexistence and conversion tools.** We know that many teams have existing infrastructure in place. We want to make it a easy as possible to choose Pulumi and, as we've worked with major customers to transition away from existing infrastructure tools, we've built tools to help ease the adoption process. This includes the following approaches:
 
-* **Coexist** with existing infrastructure, including [referencing existing Terraform remote state](/blog/using-terraform-remote-state-with-pulumi/), [deploying Kubernetes YAML as-is](https://github.com/pulumi/pulumi-kubernetes/blob/master/tests/examples/yaml-guestbook/index.ts), [deploying Kubernetes Helm charts](/blog/using-helm-and-pulumi-to-define-cloud-native-infrastructure-as-code/), [deploying AWS CloudFormation stacks](https://www.pulumi.com/docs/reference/pkg/nodejs/pulumi/aws/cloudformation/#example-usage), or [deploying Azure Resource Manager (ARM) templates](https://github.com/pulumi/examples/tree/master/azure-ts-arm-template) &mdash; in each case, enabling you to consume output properties from infrastructure deployed via other tools, for either temporary or permanent coexistence of Pulumi with these other tools.
+* **Coexist** with existing infrastructure, including [referencing existing Terraform remote state](/blog/using-terraform-remote-state-with-pulumi/), [deploying Kubernetes YAML as-is](https://github.com/pulumi/pulumi-kubernetes/blob/master/tests/examples/yaml-guestbook/index.ts), [deploying Kubernetes Helm charts](/blog/using-helm-and-pulumi-to-define-cloud-native-infrastructure-as-code/), [deploying AWS CloudFormation stacks]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/cloudformation#example-usage" >}}), or [deploying Azure Resource Manager (ARM) templates](https://github.com/pulumi/examples/tree/master/azure-ts-arm-template) &mdash; in each case, enabling you to consume output properties from infrastructure deployed via other tools, for either temporary or permanent coexistence of Pulumi with these other tools.
```

Comment:
Do the blog links in this text block also need `relref` added to them? It looks like in other places it was added, but maybe there was specific reason you avoided that here?

Reply:
Yes, these should be `relref`. I was planning to take care of these throughout in a subsequent PR.